testsuite/tests/asmcomp/func_sections.run

```diff
@@ -7,4 +7,4 @@ ${program}
 
 # now check the assembly file produced during compilation
 asm=${test_build_directory}/func_sections.s
-grep "\.section \.text\.caml\.camlFunc_sections\." "$asm" | wc -l | tr -d ' ' | sed '/^$/d'
+grep "\.section.*\.text\.caml\.camlFunc_sections\." "$asm" | wc -l | tr -d ' ' | sed '/^$/d'
```

Comment:
Can you say a word about this change?

Reply:
On s390x, the section names are emitted within double quotes, i.e.
```
.section ".text.caml.whatever",...
```
while on all other platforms, they are emitted without:
```
.section .text.caml.whatever,...
```
so this regexp change is needed for the test to pass on s390x.

Reply:
OK, thanks.

Reply:
Note that with the code factorization, text section names are no longer within quotes and this change has been reverted.